scripts/babel/transform-test-gate-pragma.js

```diff
@@ -20,8 +21,16 @@ function transform(babel) {
   // binary         →  unary ( ( "==" | "!=" | "===" | "!==" ) unary )* ;
   // unary          →  "!" primary
   //                |  primary ;
-  // primary        →  NAME | STRING | BOOLEAN
+  // primary        →  NAME | STRING | BOOLEAN | VERSION
```

Comment:
I'd prefer not to overcomplicate this DSL. Could please stick to using feature flags instead?

Reply:
Alternatively you could add a different pragma like `@react-version` instead. Then you don't have to do any fancy parsing